execution/adapter/eks_adapter.go

```diff
@@ -141,14 +142,28 @@ func (a *eksAdapter) constructAffinity(definition state.Definition, run state.Ru
 	}
 
 	if definition.Gpu == nil || *definition.Gpu <= 0 {
-		matchExpressions = append(matchExpressions, corev1.NodeSelectorRequirement{
+		requiredMatch = append(requiredMatch, corev1.NodeSelectorRequirement{
 			Key:      "beta.kubernetes.io/instance-type",
 			Operator: corev1.NodeSelectorOpNotIn,
 			Values:   gpuNodeTypes,
 		})
+
+		//For high cpu jobs - assign to c5 node types.
+		if run.Memory != nil &&
+			run.Cpu != nil &&
+			*run.Cpu > int64(0) &&
+			*run.Memory > int64(0) &&
+			float64(*run.Cpu)/float64(*run.Memory) >= 0.5 {
```

Comment:
are you going to put anything in this block?

Reply:
good catch